docs/wxm-devices/wifi-m5-bundle/m5-specs.mdx

```diff
@@ -15,7 +15,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 - Capacitive touch screen
 - ATECC608 hardware encryption
-- GPS + BLE + LoRa
+- GPS + BLE + LoRa (RF)
```

Comment:
Based on the description of the task we don't use LoRa protocol on the M5 so I suggest removing LoRa.

Reply:
yes, but the M5 generally it has a LORA stack module which we use for RF communication

Reply:
No, that's incorrect if I'm not mistaken, as the description of the Linear task says:

> The confusion most likely started because the M5 supports LoRa (and it says "LoRa" on the side where you plug the antenna) but LoRa is just one of the modulations that the M5 supports, the actual modulation used is called GFSK.

Please confirm it with the hardware team.

Reply:
Lora is the protocol, GFSK is the encryption. The M5 stack has a LORA gateway which we use for RF communication, That's why I added RF in a parenthesis..

Reply:
Hey guys :)
Both LoRa and (G)FSK are RF modulations. The M5 stack has a Sub-GHz RF transceiver that supports LoRa, GFSK and other modulations and this is what it uses to communicate with the station. However since these modulations are highly technical and are not useful for the user to know, it would be better to use a more general term "RF" eg. RF Antenna, RF transceiver, RF module, RF stack (Marked as "LoRa" on the M5 because this is how they advertise it)

Reply:
So since the above line is part of the low level specs of the WG1000 gateway it would be "- GPS + BLE + RF (LoRa, GFSK, OOK, ASK)" but this would be too much (confusing) information.
I'd say we just write it as GPS + BLE + RF (btw where's wifi? :P )

Reply:
Btw the confusion here is the typical between LoRa and LoRaWAN, where LoRaWAN is the protocol but LoRa is the modulation. However we do not use LoRaWAN nor LoRa on the M5 as our station is GFSK.